addons/markers/stringtable.xml

```diff
@@ -150,5 +150,48 @@
             <Czech>Tvůrce</Czech>
             <Turkish>Yaratıcı</Turkish>
         </Key>
+        <Key ID="STR_ACE_Markers_TimestampEnabled">
+            <English>Allow Timestamps</English>
+        </Key>
+        <Key ID="STR_ACE_Markers_TimestampEnabledDescription">
+            <English>Whether to allow timestamps to be automatically applied to markers</English>
+        </Key>
+
+        <Key ID="STR_ACE_Markers_TimestampTooltip">
+            <English>&lt;t size='0.8'&gt;Add Timestamp:&lt;/t&gt;</English>
```

Comment:
why is here styling in the stringtable?

Reply:
Skips needing to format it in script

Reply:
but may break if we have community translators that don't know what that is and maybe change something. I think styling should never be in a translation medium.